Cache/Main.lean

```diff
@@ -85,6 +92,23 @@ def main (args : List String) : IO Unit := do
   let get (args : List String) (force := false) (decompress := true) := do
     let hashMap ← if args.isEmpty then pure hashMap else hashMemo.filterByFilePaths (toPaths args)
     getFiles hashMap force force goodCurl decompress
+  let miniget := do
+    -- Find all .lean files in subdirectories (excluding .lake) of the current directory.
+    let allFiles := System.FilePath.walkDir (← IO.Process.getCurrentDir)
+      (fun p ↦ pure (p.fileName != some ".lake"))
+    let leanFiles := (← allFiles).filter (fun p ↦ p.extension == some "lean")
+    -- For each file, find all imports starting with Mathlib.
+    -- (We use proper parsing, to not be confused by lines like
+    -- 'import Mathlib -- comment' or 'import /- comment -/ Mathlib'.)
```

Comment:
I don't think we need to explain why we're calling a parsing function
```suggestion
```

Reply:
Fair point; removed.